.gitignore

```diff
@@ -1,3 +1,11 @@
 /.idea/
 /target/
 /work/
+.classpath
+.project
+.settings/
+.factorypath
+src/main/webapp/js/libs/echarts.common.min.js
```

Comment:
I'm fine with the ignores above, but can we remove the `webapp` ignores for now? Would prefer if these were addressed in a more appropriate PR

Reply:
Good point. Done.